src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,13 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+object LetterRemover {
+    @JvmStatic
+    fun removeLetter(input: String): CharArray {
```

Comment:
Internal logic could be simplified with kotlin functions
Please check this documentation
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/replace.html